modules/mbedtls/register_types.cpp

```diff
@@ -50,6 +52,8 @@ void initialize_mbedtls_module(ModuleInitializationLevel p_level) {
 		return;
 	}
 
+	GLOBAL_DEF("network/tls/enable_tls_v1.3", false);
```

Comment:
I think the `.` might be confusing the logic as that's what we use for overrides normally.

I'd suggest
```suggestion
	GLOBAL_DEF("network/tls/enable_tls_v1_3", false);
```
as a workaround.

Reply:
Ah wait, the bug I found was `allow_` vs `enable_`, it's maybe not the `.`.